### PR TITLE
Allow PHP 8

### PR DESCRIPTION
**Pull request by janlanger**
I tested it on my project and did not stumble to any issue.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
   "name": "jderusse/composer-warmup",
   "type": "composer-plugin",
   "require": {
-    "php": "^7.0",
+    "php": "^7.0 || ^8.0.0",
```
Comment (jderusse) [inline composer.json]:
```suggestion
    "php": "^7.0 || ^8.0",
```

**Review by jderusse — APPROVED** [review]

**Comment by jderusse** [comment]
THank you

**Comment by holtkamp** [comment]
Nice, would this justify tagging a new release?

**Comment by jderusse** [comment]
> Nice, would this justify tagging a new release?

sure! 
Done

**Pull request merged**